lib/tasks/db.rake

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+namespace :db do
```

Comment:
This is the main change I can see that could affect non-Nanobox users/functionality.

Can you expand on the methodology behind this? Looks like it's overriding the default `db:migrate`